app/assets/javascripts/miq_dynatree.js

```diff
@@ -321,6 +321,15 @@ function cfmeOnClick_GenealogyTree(id) {
   }
 }
 
+// OnCheck handler for the SmartProxy Affinity tree
+function cfmeOnClick_SmartProxyAffinityCheck(node) {
```

Comment:
@chessbyte to be consistent with other functions in that file, kept the cfme prefix.